firefox_puppeteer/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Permissions(BaseLib):
+    def remove(self, host, permission):
+        with self.marionette.using_context('chrome'):
```

Comment:
We don't seem to be in chrome context when this function gets run. It fails without the line setting context, like so:

avascriptException: ReferenceError: Cu is not defined
stacktrace:
	execute_script @utils.py, line 21
	inline javascript, line 1
	src: "          Cu.import("resource://gre/modules/Services.jsm");"